tests/job_tests/recognition/test_scoring.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+from typing import Optional
+
+from sisyphus import tk, setup_path
+
+from i6_core.tools.compile import MakeJob
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.recognition import ScliteJob
+
+Path = setup_path(__package__)

```

Comment:
If you want to keep using this (see separate thread), then this should be:
```suggestion
rel_path = setup_path(__package__)
```